### PR TITLE
[libc++] Upgrade to GCC 14

### PR DESCRIPTION
**Pull request by philnik777**

### CODE REVIEW

File: libcxx/trigger
Comment (ldionne) [inline libcxx/trigger:1]: I think this isn't necessary since you're modifying files inside `libcxx/` anyway.

**Review by ldionne — APPROVED** [review]
LGTM, but we also need to update the documentation that mentions which version of GCC we support.

File: libcxx/trigger
Comment (philnik777) [reply libcxx/trigger:1]: We have the file in tree currently. This removes it.

Comment (ldionne) [reply libcxx/trigger:1]: Crap, I think that must be my fault. Thanks for removing it.

**Pull request merged**